src/optimal_retention.rs

```diff
@@ -1331,57 +1339,61 @@ mod tests {
                 Ok(())
             };
 
+        let wrap = |f: Box<
+            (dyn for<'a> Fn(&'a Card) -> i32 + std::marker::Send + std::marker::Sync + 'static),
+        >| Some(ReviewPriorityFn(Arc::new(f)));
+
```

Comment:
idk but this seems more clear.

```suggestion
        macro_rules! wrap {
             ($f:expr) => {Some(ReviewPriorityFn(std::sync::Arc::new($f)))};
        }
```

Reply:
see https://github.com/open-spaced-repetition/fsrs-rs/pull/288/commits/0085c7ff9134b4ec12693475dc88ea6086c369ae as an example :)